jobs.ts

```diff
@@ -0,0 +1,57 @@
+import { assert, CosmWasmClient, Uint53 } from "./deps.ts";
+import { chainHash, timeOfRound } from "./drand.ts";
+
+interface Job {
```

Comment:
Curious question, Is this wasmable ?

Reply:
Not sure what you means, but this is the JSON format of the jobs returned from the gateway. We want the type here, not any executable logic. There is a JSON Schema for that which the contract can create but JSON Schema -> TypeScript conversion is not that easy to set up.

Reply:
Okay makes sense. I thought wasm (apart from porting function execution ) could also help to just export some struct or enum not only functions, such that people would prevent deduplication of interfaces and APIs between two different  languages. This is irrelevant to this PR anyways.